http-server/src/main/java/io/airlift/http/server/ReloadableSslContextFactoryProvider.java

```diff
@@ -218,24 +218,24 @@ private static KeyStore loadKeyStore(Optional<File> keystoreFile, String keystor
         }
     }
 
-    private static KeyStore loadTrustStore(File truststoreFile, String truststorePassword)
+    private static KeyStore loadTrustStore(File trustStoreFile, String trustStorePassword)
```

Comment:
Unrelated change? Move to a separate commit.

Reply:
related, inconsistent naming of variables